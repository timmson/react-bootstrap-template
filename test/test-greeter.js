const React = require("react");
const renderer = require("react-test-renderer");

const Greeter = require("../src/greeter");

class MockCalc {

	hash() {
		return 0;
	}

}

describe("Greeter should", () => {

	test("return greeting", () => {
		const component = renderer.create(
			<Greeter calc={new MockCalc()}/>
		);
		expect(component.toJSON()).toMatchSnapshot();
		component.unmount();
	});
});