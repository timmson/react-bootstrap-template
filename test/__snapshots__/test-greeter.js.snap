// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeter should return greeting 1`] = `
<div
  className="container"
  id="app"
>
  <h2
    className="text-center"
  >
    Your fist React+Bootstrap app
  </h2>
  <div
    className="row"
  >
    <div
      className="col-4 text-right pt-4"
    >
      <p>
        Your name:
      </p>
    </div>
    <div
      className="col align-middle"
    >
      <input
        className="form-control mt-4"
        id="username"
        onKeyUp={[Function]}
        placeholder="Enter your name ..."
        type="text"
      />
    </div>
  </div>
  <div
    className="row mt-3"
  >
    <div
      className="col-3"
    >
      Â 
    </div>
    <div
      className="col"
    >
      <p
        className="my-style mt-2"
      >
        
      </p>
    </div>
  </div>
</div>
`;
